odk1-src/central-intro.rst

```diff
@@ -42,9 +42,13 @@ See `What is coming in Central <https://forum.opendatakit.org/t/whats-coming-in-
 Who should use ODK Central?
 ---------------------------
 
-Central is still early in its development and is missing many features that some more complicated projects will rely on (see the above list). But we do feel good about the features that we do already support.
+Central is much younger than Aggregate. Because of this, it isn't as heavily battle-tested, and you might come across quirks and problems while using it. Among other things, it can be a little tricky to install. But the issues people have run into have been minor so far, and we haven't yet heard of anybody losing any data. As long as you :doc:`enable backups <central-backup>`, we feel good about the reliability and integrity of your data. Part of this comes from how careful Central is about handling your uploaded data.
```

Comment:
>  Among other things, it can be a little tricky to install, but the issues people have run into have been minor so far.

We don't need to put data loss in people's minds, especially since we are confident it's unlikely.

Reply:
i feel like it's like the first thing that people are going to wonder?

Reply:
My guess is that people trust the brand and assume it won't lose data. I think the first thing they wonder is more if it has all the features they need. I don't fee strongly about this. Your call.

Reply:
okay, i'll remove it.